web/turkey/survey/static/survey/js/auditors/keypresses_total.js

```diff
@@ -0,0 +1,25 @@
+var AuditorKeypressesTotal = {
+    keypresses_total: 0,
+    log_keypress_event: function (e) {
+        // e.stopPropagation();
```

Comment:
In the original auditors.js when I was testing the code, pressing a key would log it twice. That's why I had e.stopPropagation(). I used a different keypress event detection in that file, so unsure if that problem pertains to this file since it has not yet been tested. That's why it's commented out.